tests/modules/programs/firefox/profile-settings.nix

```diff
@@ -15,13 +15,30 @@ lib.mkIf config.test.enableBig {
     profiles.bookmarks = {
       id = 2;
       settings = { "general.smoothScroll" = false; };
-      bookmarks = {
-        wikipedia = {
+      bookmarks = [
```

Comment:
I think that both test cases should be included to ensure that the old bookmarks-only attribute set structure for the `bookmarks` option continues to work.

Reply:
That makes sense. I'll add it back in.